EventFilter/EcalRawToDigi/src/MatacqRawEvent.cc

```diff
@@ -137,7 +136,7 @@ void MatacqRawEvent::setRawData(const unsigned char* pData, size_t maxSize){
   const int nCh = getChannelCount();
   channelData.resize(nCh);
 
-  pData16 = (int16le_t*) (begin+headerLen); 
+  pData16 = (int16le_t*) (pData+headerLen); 
```

Comment:
The static analyzer is still finding problems. `pData16` should be defined here on line 139:
`const int16le_t *pData16 = (const int16le_t *) (pData+headerLen); `
Then delete line 85, and also lines 91 and 92, which has an obsolete comment.